### ISSUE TITLE
star marker invisible in latex output

### ISSUE THREAD

**Issue by ferdymercury** [root-project/root#9143]
- [x] Checked for duplicates

### Describe the bug
When using TH1::Draw("P*"), the PNG and PDF output looks fine, but the markers are invisible in the TEX output

![image](https://user-images.githubusercontent.com/10653970/137940738-13f9dee9-53c5-4984-964f-ce9740e4d681.png)

### Expected behavior
Markers are visible in TEX also.

### To Reproduce
from ROOT import TCanvas, TH1D
```
c = TCanvas()
h = TH1D("h", "h", 10,0,10)
h.SetBinContent(2,10)
h.Draw("P*")
c.SaveAs("test.png")
c.SaveAs("test.pdf")
c.SaveAs("test.tex")
```
and then compile the latex output using:

```
\documentclass{standalone}
\usepackage{tikz}
\begin{document}
\input{test.tex}
\end{document}
```

### Setup
```
   ------------------------------------------------------------------
  | Welcome to ROOT 6.25/01                        https://root.cern |
  | (c) 1995-2021, The ROOT Team; conception: R. Brun, F. Rademakers |
  | Built for linuxx8664gcc on Sep 01 2021, 23:19:42                 |
  | From heads/master@v6-25-01-1861-g6b33e04cac                      |
  | With c++ (Ubuntu 8.4.0-1ubuntu1~18.04) 8.4.0                     |
  | Try '.help', '.demo', '.license', '.credits', '.quit'/'.q'       |
   ------------------------------------------------------------------
```

### Additional context
https://sft.its.cern.ch/jira/browse/ROOT-8587

**Comment by couet** [comment]
I went back to https://github.com/root-project/root/blob/f692b05a07a25a92447db3e3428726285e61ff35/graf2d/postscript/src/TTeXDump.cxx (2013)
And it is still the same ... I am trying to understand ....

**Comment by couet** [comment]
and see also:
https://root.cern/blog/saving-canvas-in-TeX/

**Comment by couet** [comment]
I tried a bit to test it the library `patterns` is loaded or nor.
I added the line 

```
\iftikzlibraryloaded{patterns}{}{}
```

at the beginning of the TeX/tikz file generated by ROOT.
I tried several different things but, when I run `pdflatex`, on the TeX file including the picture I always get:

```
! Undefined control sequence.
l.1 \iftikzlibraryloaded
                        {patterns}{}{}
? 
```
I tried several different things, without success... I guess I will not spend more time on it.
